shopBuddy/shopBuddy/VCs/CartViewController.m

```diff
@@ -31,12 +32,33 @@ - (void)viewDidLoad {
     tableView.delegate = self;
     tableView.rowHeight = UITableViewAutomaticDimension;
     manager = [AppState sharedManager];
-    [self _updateView];
+    CGRect screenRect = [[UIScreen mainScreen] bounds];
+    __weak __typeof__(self) weakSelf = self;
+    [UIView animateWithDuration:0.2 animations:^{
+        self->animationView.frame = CGRectMake(screenRect.size.width, animationView.frame.origin.y, animationView.frame.size.width, animationView.frame.size.height);
+    } completion:^(BOOL finished) {
+        [weakSelf _updateView:^(BOOL succeeded) {
+            dispatch_async(dispatch_get_main_queue(), ^{
```

Comment:
Read comment above. Move dispatch_async to main queue to private method and reuse.